frontend/src/components/Card.tsx

```diff
@@ -35,7 +44,8 @@ export default function Card(props: CardProps) {
                 props.me !== "anonymousUser" ?
                     <div className="cardContainerFooter">
                         <button className="cardBtn favouriteBtn"
-                                onClick={() => props.addItem(props.me, props.filteredNasaData.explanation, props.filteredNasaData.title, props.filteredNasaData.url)}>
+                                style={{'display': `${props.favouriteBtnDisplay ? 'inline' : 'none'}`}}
+                                onClick={handleAddToFavourite}>
                             <MdFavorite/>
                             <span className="btnHoverText">Add to favourite</span>
```

Comment:
I think, it would have been easier/shorter, if you just added another property to item like 
"likedByUser":[user1,user153,user342]. Then you could have filtered the favorites of a user by searching his id in the likedBy-property of every item.... 
Or... giving the user an additional property like "favorites":[item3,item7,item456] and then only render the ones with those id's....
Then you wouldn't have to create a whole Favorite-List for each user. But as you had all the effort to write it like that - maybe you'd like to try that in your next project. 🙃

Reply:
It´s good idea, I think I will do it. Thanks